tsup.config.ts

```diff
@@ -30,4 +30,7 @@ export default defineConfig({
   env: {
     PACKAGE_NAME: pkgJson.name,
   },
+  loader: {
+    '.js': 'jsx',
+  },
```

Comment:
Added this after switching `styled.jsx` to `styled.js` to make the build work. I can't run the test with `.jsx` and there is no `.jsx` found the codebase.